backends/events/sdl/sdl-events.cpp

```diff
@@ -900,8 +938,10 @@ bool SdlEventSource::remapKey(SDL_Event &ev, Common::Event &event) {
 void SdlEventSource::resetKeyboardEmulation(int16 x_max, int16 y_max) {
 	_km.x_max = x_max;
 	_km.y_max = y_max;
-	_km.delay_time = 25;
+	_km.delay_time = 12;
```

Comment:
Ok it is fixed, see last commit and my comment. the factor "delay/25" assures the pointer velocity is independent of delay, and will be the same as it was before this PR. But with a much smoother pointer movement.